mnist/mnist_gcp.ipynb

```diff
@@ -1717,37 +1759,39 @@
    "source": [
     "## Access the  web UI\n",
     "\n",
-    "* A reverse proxy route is automatically added to the Kubeflow IAP endpoint\n",
-    "* The endpoint will be\n",
+    "A reverse proxy route is automatically added to the Kubeflow IAP endpoint. The endpoint is:\n",
     "\n",
     "  ```\n",
     "  http:/${KUBEflOW_ENDPOINT}/mnist/${NAMESPACE}/ui/  \n",
-    "  ```kubeflow-jlewi\n",
-    "* You can get the KUBEFLOW_ENDPOINT\n",
+    "  ```\n",
+    "\n",
+    "You can run the following command to get the KUBEFLOW_ENDPOINT:\n",
     "\n",
     "  ```\n",
-    "  KUBEfLOW_ENDPOINT=`kubectl -n istio-system get ingress envoy-ingress -o jsonpath=\"{.spec.rules[0].host}\"`\n",
+    "  KUBEFLOW_ENDPOINT=`kubectl -n istio-system get ingress envoy-ingress -o jsonpath=\"{.spec.rules[0].host}\"`\n",
     "  ```\n",
     "  \n",
-    "  * You must run this command with sufficient RBAC permissions to get the ingress.\n",
-    "  \n",
-    "* If you have sufficient privileges you can run the cell below to get the endpoint if you don't have sufficient priveleges you can \n",
-    "  grant appropriate permissions by running the command\n",
+    "You must run the above command with sufficient RBAC permissions to get the ingress. \n",
+    "\n",
+    "* If you have sufficient privileges you can run the cell below to get the endpoint. \n",
+    "* If you don't have sufficient priveleges you can grant appropriate permissions by running the following command:\n",
     "  \n",
     "   ```\n",
     "   kubectl create --namespace=istio-system rolebinding --clusterrole=kubeflow-view --serviceaccount=${NAMESPACE}:default-editor ${NAMESPACE}-istio-view\n",
```

Comment:
@jlewi is this the kubectl command that you mentioned, where the `NAMESPACE` formatting is weird? I haven't actually changed this command. Maybe it's the comment I added below, where `NAMESPACE` is in code formatting? It looks OK in my notebook, so I'm not sure which occurrence of NAMESPACE looks weird in your notebook.

Reply:
Sorry it wasn't due to your changes; it was an existing issue that I just happened to notice.